src/structures/Typing.js

```diff
@@ -7,11 +7,6 @@ const Base = require('./Base');
  * @extends {Base}
  */
 class Typing extends Base {
-  /**
-   * @param {TextBasedChannels} channel The channel this typing came from
-   * @param {User} user The user that started typing
-   * @param {APITypingStart} data The raw data received
```

Comment:
This was the last instance of `APITypingStart`. Should we remove this too?

https://github.com/discordjs/discord.js/blob/395186da7241b1ea704e4d5e0c6df6fd4655eb5c/src/structures/Typing.js#L74-L77